### ISSUE TITLE
Move to JuliaGraphics caused missing Qt plugin?

### ISSUE THREAD

**Issue by veddox** [JuliaGraphics/QML.jl#171]
Hi everyone,

I've just started using QML.jl, and am completely new to Qt & QML, so I may be completely wrong here. But is it possible that the move to JuliaGraphics broke something with the package?

I've been running QML.jl from the main branch, as recommended [here](https://github.com/JuliaGraphics/QML.jl/issues/168#issuecomment-1694631305). Things were fine until yesterday afternoon (CEST), when I reinstalled the package, and since then nothing has been working. I always get the following error when importing QML.jl:

```
julia> using QML
qt.qpa.plugin: Could not find the Qt platform plugin "xcb" in ""
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: linuxfb, offscreen, eglfs, minimalegl, vnc, minimal, vkkhrdisplay.
```
<details><summary>Full error message</summary>
<code>
julia> using QML
qt.qpa.plugin: Could not find the Qt platform plugin "xcb" in ""
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: linuxfb, offscreen, eglfs, minimalegl, vnc, minimal, vkkhrdisplay.

[76092] signal (6.-6): Aborted
in expression starting at REPL[4]:1
pthread_kill at /lib/x86_64-linux-gnu/libc.so.6 (unknown line)
raise at /lib/x86_64-linux-gnu/libc.so.6 (unknown line)
abort at /lib/x86_64-linux-gnu/libc.so.6 (unknown line)
_Z6qAbortv at /home/user/.julia/artifacts/0a970ac8426a9e414db626eb96c788b0d592369f/lib/libQt6Core.so.6 (unknown line)
_ZNK14QMessageLogger5fatalEPKcz at /home/user/.julia/artifacts/0a970ac8426a9e414db626eb96c788b0d592369f/lib/libQt6Core.so.6 (unknown line)
_ZL13init_platformRK7QStringS1_S1_RiPPc.cold at /home/user/.julia/artifacts/0a970ac8426a9e414db626eb96c788b0d592369f/lib/libQt6Gui.so (unknown line)
_ZN22QGuiApplicationPrivate25createPlatformIntegrationEv at /home/user/.julia/artifacts/0a970ac8426a9e414db626eb96c788b0d592369f/lib/libQt6Gui.so (unknown line)
_ZN22QGuiApplicationPrivate21createEventDispatcherEv at /home/user/.julia/artifacts/0a970ac8426a9e414db626eb96c788b0d592369f/lib/libQt6Gui.so (unknown line)
_ZN23QCoreApplicationPrivate4initEv at /home/user/.julia/artifacts/0a970ac8426a9e414db626eb96c788b0d592369f/lib/libQt6Core.so.6 (unknown line)
_ZN22QGuiApplicationPrivate4initEv at /home/user/.julia/artifacts/0a970ac8426a9e414db626eb96c788b0d592369f/lib/libQt6Gui.so (unknown line)
_ZN15QGuiApplicationC1ERiPPci at /home/user/.julia/artifacts/0a970ac8426a9e414db626eb96c788b0d592369f/lib/libQt6Gui.so (unknown line)
_ZNSt17_Function_handlerIFN5jlcxx10BoxedValueI15QGuiApplicationEERiPPcEZNS0_6Module11constructorIS2_JS4_S6_EEEvP14_jl_datatype_tbEUlS4_S6_E_E9_M_invokeERKSt9_Any_dataS4_OS6_ at /home/user/.julia/artifacts/f1b409a622d1a93db281f51631bc9dbcd42c72bc/lib/libjlqml.so (unknown line)
_ZN5jlcxx6detail11CallFunctorINS_10BoxedValueI15QGuiApplicationEEJRiPPcEE5applyEPKvNS_13WrappedCppPtrESB_ at /home/user/.julia/artifacts/f1b409a622d1a93db281f51631bc9dbcd42c72bc/lib/libjlqml.so (unknown line)
QGuiApplication at /home/user/.julia/packages/CxxWrap/aXNBY/src/CxxWrap.jl:624
unknown function (ip: 0x7fd541ee92c6)
_jl_invoke at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/gf.c:2758 [inlined]
ijl_apply_generic at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/gf.c:2940
__init__ at /home/user/.julia/packages/QML/dFnTL/src/QML.jl:151
jfptr___init___1862 at /home/user/.julia/compiled/v1.9/QML/JLkMo_p2KP2.so (unknown line)
_jl_invoke at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/gf.c:2758 [inlined]
ijl_apply_generic at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/gf.c:2940
jl_apply at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/julia.h:1880 [inlined]
jl_module_run_initializer at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/toplevel.c:75
ijl_init_restored_modules at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/module.c:982
register_restored_modules at ./loading.jl:1115
_include_from_serialized at ./loading.jl:1061
_require_search_from_serialized at ./loading.jl:1506
_require at ./loading.jl:1783
_require_prelocked at ./loading.jl:1660
macro expansion at ./loading.jl:1648 [inlined]
macro expansion at ./lock.jl:267 [inlined]
require at ./loading.jl:1611
jfptr_require_45889.clone_1 at /usr/local/lib/julia/lib/julia/sys.so (unknown line)
_jl_invoke at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/gf.c:2758 [inlined]
ijl_apply_generic at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/gf.c:2940
jl_apply at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/julia.h:1880 [inlined]
call_require at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/toplevel.c:466 [inlined]
eval_import_path at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/toplevel.c:503
jl_toplevel_eval_flex at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/toplevel.c:731
jl_toplevel_eval_flex at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/toplevel.c:856
ijl_toplevel_eval_in at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/toplevel.c:971
eval at ./boot.jl:370 [inlined]
eval_user_input at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/usr/share/julia/stdlib/v1.9/REPL/src/REPL.jl:153
repl_backend_loop at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/usr/share/julia/stdlib/v1.9/REPL/src/REPL.jl:249
#start_repl_backend#46 at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/usr/share/julia/stdlib/v1.9/REPL/src/REPL.jl:234
start_repl_backend at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/usr/share/julia/stdlib/v1.9/REPL/src/REPL.jl:231
_jl_invoke at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/gf.c:2758 [inlined]
ijl_apply_generic at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/gf.c:2940
#run_repl#59 at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/usr/share/julia/stdlib/v1.9/REPL/src/REPL.jl:379
run_repl at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/usr/share/julia/stdlib/v1.9/REPL/src/REPL.jl:365
jfptr_run_repl_60908.clone_1 at /usr/local/lib/julia/lib/julia/sys.so (unknown line)
_jl_invoke at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/gf.c:2758 [inlined]
ijl_apply_generic at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/gf.c:2940
#1017 at ./client.jl:421
jfptr_YY.1017_36106.clone_1 at /usr/local/lib/julia/lib/julia/sys.so (unknown line)
_jl_invoke at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/gf.c:2758 [inlined]
ijl_apply_generic at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/gf.c:2940
jl_apply at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/julia.h:1880 [inlined]
jl_f__call_latest at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/builtins.c:774
#invokelatest#2 at ./essentials.jl:819 [inlined]
invokelatest at ./essentials.jl:816 [inlined]
run_main_repl at ./client.jl:405
exec_options at ./client.jl:322
_start at ./client.jl:522
jfptr__start_40034.clone_1 at /usr/local/lib/julia/lib/julia/sys.so (unknown line)
_jl_invoke at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/gf.c:2758 [inlined]
ijl_apply_generic at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/gf.c:2940
jl_apply at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/julia.h:1880 [inlined]
true_main at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/jlapi.c:573
jl_repl_entrypoint at /cache/build/default-amdci5-5/julialang/julia-release-1-dot-9/src/jlapi.c:717
main at julia (unknown line)
unknown function (ip: 0x7fd543229d8f)
__libc_start_main at /lib/x86_64-linux-gnu/libc.so.6 (unknown line)
unknown function (ip: 0x4010b8)
Allocations: 4843227 (Pool: 4838808; Big: 4419); GC: 8
fish: Job 1, 'julia' terminated by signal SIGABRT (Abort)
</code>
</details> 

At first I thought this was a local problem, but I've set up a fresh Julia+QML.jl install on a different Linux machine, and gotten the same crash. Is it at all possible that this is related to the migration? If not, any ideas what might cause this?

**Comment by barche** [comment]
No, this is the upgrade to Qt6.5 in Yggdrasil, downgrading Qt6Base_jll to 6.4.2 should work around this until I can rebuild the package.

**Comment by veddox** [comment]
Oh, wonderful - thank you for the quick response! Yes, that fixed it.

I suggest leaving this issue open until the package has been rebuilt, in case others run into the same problem.

**Comment by ufechner7** [comment]
Perhaps this can be closed now that 0.8 is released?
